readme.md

```diff
@@ -414,7 +414,7 @@ Other awesome open source projects that use Lighthouse.
 * **[webpack-lighthouse-plugin](https://github.com/addyosmani/webpack-lighthouse-plugin)** - Run Lighthouse from a Webpack build.
 * **[cypress-audit](https://github.com/mfrachet/cypress-audit)** - Run Lighthouse and Pa11y audits directly in your E2E test suites.
 * **[laravel-lighthouse](https://github.com/adityadees/laravel-lighthouse)** - Google Lighthouse wrapper for laravel framework to run Google Lighthouse CLI with custom option and can automatically save result in your server directory.
-
+* **[Neodymium](https://github.com/Xceptance/neodymium/wiki/Reports#use-lighthouse-inside-neodymium)** - The test automation framework Neodymium integrates Lighthouse for accessibility and Web Vitals verification. You can programmatically validate and assert all audit values.
```

Comment:
The "Use Lighthouse inside Neodymium" section doesn't seem to exist on this link. Has the documentation moved?

Reply:
You are correct, a documentation rework moved this. I updated the PR with the latest link. I am sorry about the oversight.